src/components/views/Header.js

```diff
@@ -93,6 +101,25 @@ function Header(props){
             </div>
         }
     }
+    if(props.view==="userPage"){
+        return <div className="header container" style={{height:"100"}}>
+            <h1 className="header title">Date Against Humanity</h1>
+            <div className="header button-container">
+                <Button
+                    onClick={() => logout()}
+                >
+                    Logout
+                </Button>
+            </div>
+            <div className="header button-container-gameMenu">
+                <Button
+                    onClick={() => goToGameMenu()}
+                >
+                    Game Menu
```

Comment:
Game Menu -> Menu ?

Reply:
In the current version we called the view "Game Menu", the overview screen with the white card
![Screenshot 2022-04-08 134040](https://user-images.githubusercontent.com/91139250/162428741-23ac90ca-4113-48f3-ba46-5da53953115d.png)
s

Reply:
I mean the Button Name shouldn't be called Game Menu but rather just Menu

Reply:
done by joe